functions/adjustChatCapacity.ts

```diff
@@ -14,33 +14,40 @@ import {
 
 const TokenValidator = require('twilio-flex-token-validator').functionValidator;
 
-type EnvVars = {};
+type EnvVars = {
+  TWILIO_WORKSPACE_SID: string;
+};
 
 export type Body = {
   workerSid?: string;
-  workspaceSid?: string;
-  channelSid?: string;
-  workerLimit?: number;
   adjustment?: 'increase' | 'decrease';
 };
 
-const adjustChatCapacity = async (
+export const adjustChatCapacity = async (
   context: Context<EnvVars>,
   body: Required<Body>,
 ): Promise<{ status: number; message: string }> => {
   const client = context.getTwilioClient();
 
-  const channel = await client.taskrouter
-    .workspaces(body.workspaceSid)
+  const worker = await client.taskrouter
+    .workspaces(context.TWILIO_WORKSPACE_SID)
     .workers(body.workerSid)
-    .workerChannels(body.channelSid)
     .fetch();
 
+  if (!worker) return { status: 404, message: 'Could not find worker.' };
```

Comment:
Will the error here and at L44 be logged in Twilio debugger or in Rollbar (via the Flex plugin) so that we will know about them?  By moving the worker and channel lookup to this function, we are creating more remote API calls, and more things that can fail.  That is okay if we have a way to detect when that happens.

Reply:
Yes. I intentionally triggered that (setting the condition to true). When we manage to connect Rollbar to debugger hook, this will be fired as an event, and it's still in Twilio's console in the meanwhile.
![Screenshot from 2020-10-07 16-32-25](https://user-images.githubusercontent.com/15805319/95378856-e7278c00-08ba-11eb-8f69-718c59a61c9a.png)